indico/modules/events/abstracts/controllers/base.py

```diff
@@ -152,7 +152,8 @@ class AbstractsExportPDFMixin:
 
     def _process(self):
         sorted_abstracts = sorted(self.abstracts, key=attrgetter('friendly_id'))
-        pdf = ConfManagerAbstractsToPDF(self.event_new, sorted_abstracts)
+        cls = ConfManagerAbstractsToPDF if self.management else AbstractsToPDF
```

Comment:
Yeah... if someone has a use case to download "full" PDFs e.g. as a convener we can always make this more sophisticated. I don't think we ever need per-abstract here, if you aren't in management you only get abstracts from one track here (but we'd need to enforce this when using the management pdf).